imageio/imageio-psd/src/main/java/com/twelvemonkeys/imageio/plugins/psd/PSDLayerInfo.java

```diff
@@ -33,7 +33,7 @@
 import javax.imageio.IIOException;
 import javax.imageio.stream.ImageInputStream;
 import java.io.IOException;
-import java.util.Arrays;
+import java.util.*;
```

Comment:
This file does not follow the code style. Please reformat the code according the project's code style.

* Prefer single class imports
* Use space after keywords (`if`, `for`, `while` etc)
* Use space before curly brace
* Use `else` on separate line